lib/reference/prefixes.js

```diff
@@ -0,0 +1,5 @@
+const prefixes = new Set(['-webkit-', '-moz-', '-ms-', '-o-']);
```

Comment:
If this is to be reused, Ill eventually add some more rare ones to the set.

ref: https://stackoverflow.com/questions/11705144/webkit-and-its-legacy-prefixes